From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Fri, 7 Feb 2020 14:34:50 -0800
Subject: ANDROID: gki_defconfig: Relocate INCREMENTAL_FS

Ensure we build incrementalfs.ko as a module. As GKI is not ready for
module artifacts, move this to the board fragments.

Bug: 133435829
Change-Id: Ice50969b49b437606a0d9fd64844c8cfff05536e
Signed-off-by: Alistair Delva <adelva@google.com>
[joneslee: Removed defconfig elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 arch/arm64/configs/db845c_gki.fragment | 1 +
 cuttlefish.fragment                    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -60,6 +60,7 @@ CONFIG_QCOM_QFPROM=m
 CONFIG_INTERCONNECT_QCOM=y
 CONFIG_INTERCONNECT_QCOM_OSM_L3=m
 CONFIG_INTERCONNECT_QCOM_SDM845=m
+CONFIG_INCREMENTAL_FS=m
 CONFIG_WLAN_VENDOR_ATH=y
 CONFIG_ATH10K_AHB=y
 CONFIG_ATH10K=m
diff --git a/cuttlefish.fragment b/cuttlefish.fragment
--- a/cuttlefish.fragment
+++ b/cuttlefish.fragment
@@ -16,3 +16,4 @@ CONFIG_VIRTIO_PMEM=m
 CONFIG_VIRTIO_INPUT=m
 CONFIG_VIRTIO_MMIO=m
 CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
+CONFIG_INCREMENTAL_FS=m
